src/components/area.js

```diff
@@ -88,7 +103,7 @@ const Area = ({ classes, size, inputProps, label, maxLength, helperText, disable
       </form>
       <div className="grid grid-cols-2">
         {helperText && <p className={areaClasses.helperText}>{helperText}</p>}
-        {!isNaN(maxLength) && <p className={areaClasses.wordCount}>{value.length || 0}/{maxLength}</p>}
+        {!isNaN(maxLength) && <p className={areaClasses.wordCount}>{_value && _value.length || 0}/{maxLength}</p>}
```

Comment:
I would use optional chaining here` _value?.length` instead of  `_value && _value.length`